### PR TITLE
Related entities

### PR DESCRIPTION
**Pull request by rtmill**
-display entities related to individuals

### CODE REVIEW

File: app/domain/membership/service.js [hunk 1]
```diff
@@ -21,6 +30,34 @@ function MembershipService(EntityService) {
...
+        return $q(function(resolve, reject) {
+            responsePromise.then(
+                function(response) {
+                    console.log(response);
+                    var entities = response.data;
+                    resolve(entities);
+                },
+                function(response){
+                    console.log(response);
+                    window.alert("Status: " + response.status +
```
Comment (munk) [inline app/domain/membership/service.js]: this should use the errorHandler function

File: app/domain/relationship/service.js [hunk 2]
```diff
@@ -21,6 +33,33 @@ function RelationshipService(EntityService) {
...
+
+
+        return $q(function(resolve, reject) {
+            responsePromise.then(
+                function(response) {
+                    console.log(response);
+                    var entities = response.data;
+                    resolve(entities);
+                },
+                function(response){
```
Comment (munk) [inline app/domain/relationship/service.js]: this should use the errorHandler function

File: app/domain/residency/service.js [hunk 3]
```diff
@@ -2,12 +2,45 @@
...
+
+
+        return $q(function(resolve, reject) {
+            responsePromise.then(
+                function(response) {
+                    console.log(response);
+                    var entities = response.data;
+                    resolve(entities);
+                },
+                function(response){
```
Comment (munk) [inline app/domain/residency/service.js]: this should use the errorHandler function

File: app/domain/visitEvent/service.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+        return $q(function(resolve, reject) {
+            responsePromise.then(
+                function(response) {
+                    console.log(response);
+                    var entities = response.data;
+                    resolve(entities);
+                },
+                function(response){
```
Comment (munk) [inline app/domain/visitEvent/service.js]: This should use the errorHandler function

**Review by munk — CHANGES_REQUESTED** [review]
This looks good, but we should use the errorHandler function instead of an alert box

**Comment by rtmill** [comment]
made requested adjustments

**Review by munk — APPROVED** [review]

**Pull request merged**